.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+on: [push]
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the source code
+      uses: actions/checkout@v2
+    - name: Validate SQF
+      run: python3 tools/sqf_validator.py
+    - name: Validate Config
+      run: python3 tools/config_style_checker.py
+    - name: Check for BOM
+      uses: arma-actions/bom-check@master
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the source code
+      uses: actions/checkout@master
+    - name: Lint (sqflint)
+      uses: jokoho48/sqflint@master
+      continue-on-error: true # No failure due to many false-positives
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Build addon with HEMTT
+        uses: arma-actions/hemtt@master
+        with:
+          command: 'build --release'
+      - uses: actions/upload-artifact@master
+        with:
+          name: '@ZEN'
```

Comment:
I don’t know but this will already prefix it but it does need the version number.

Reply:
Upload artifact is really rudimentary, you can't even properly upload a zip without double-zipping it. Attaching a proper version is its own issue. Environment variables maybe? Ping me if you make if work! 

I commented it out in a few of my projects.

Reply:
@jonpas V2 of upload-artifact will support Glob wildcards, haven't figured out how to not have it double-zip though.

Reply:
Yeah, I am still waiting for that.